src/cpuinfo_aarch64.c

```diff
@@ -113,29 +116,15 @@ int GetAarch64FeaturesEnumValue(const Aarch64Features* features,
     case AARCH64_CRC32:
       return features->crc32;
     case AARCH64_LAST_:
+    default:
```

Comment:
Remove the default case here and in all the switches to allow the compiler to warn about missing enum case.

Reply:
fixed